website/docs/r/alert_configuration.html.markdown

```diff
@@ -119,6 +119,22 @@ resource "mongodbatlas_alert_configuration" "test" {
 }
 ```
 
+### Create third party notification using credentials from existing third party integration
+
+
+```terraform
+resource "mongodbatlas_alert_configuration" "test" {
+  project_id = "PROJECT ID"
+  enabled    = true
+  event_type = "USERS_WITHOUT_MULTI_FACTOR_AUTH"
+
+  notification {
+    type_name     = "PAGER_DUTY"
+    integration_id = "THIRD PARTY INTEGRATION ID"
```

Comment:
Our [third_party_integration data source](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/data-sources/third_party_integration) currently is not capable of obtaining this id (while it is being returned in the API), will create a follow up ticket to address this.